web/src/engine/osk/src/input/gestures/specsForLayout.ts

```diff
@@ -31,10 +31,16 @@ export interface GestureParams<Item = any> {
      */
     permitsFlick: (item?: Item) => boolean,
 
+    /**
+     * The minimum _net_ distance traveled before a longpress flick-shortcut will cancel any
+     * conflicting flick models.
```

Comment:
To be clear, it only does so if there are no actual northish flicks.  (nw, n, ne)